.github/docker-performance-tests/docker-compose.yml

```diff
@@ -37,7 +37,7 @@ services:
       - AWS_DEFAULT_REGION
       - SAMPLE_APP_LOG_LEVEL=ERROR
       - OTEL_RESOURCE_ATTRIBUTES=service.name=aws-otel-integ-test
-      - OTEL_EXPORTER_OTLP_ENDPOINT=http://otel:4317
+      - OTEL_EXPORTER_OTLP_ENDPOINT=http://otel:4318
```

Comment:
any specific use case to modifying the port to 4318? pls

Reply:
The [OpenTelemetrySpecifications for HTTP exporters](https://github.com/open-telemetry/opentelemetry-specification/blob/b7473b5de0f55f921f896948442ebb274f58b584/specification/protocol/exporter.md#configuration-options) says that the default port to export to is `4318`. Our app needs to adjust to this because in `collector-config.yml` we use the `http` receiver which is listening on that port. (Not `4317` like the gRPC exporter/receiver).